plugins/extractit/plugin.spec.yaml

```
@@ -0,0 +1,292 @@
+plugin_spec_version: v2
+extension: plugin
+products: ["insightconnect"]
+name: extractit
+title: ExtractIt
+description: The ExtractIt plugin has a collection of actions used to extract various information from text. These include URLs, emails, and IPs
```

Minor thing I noticed in the old version. Update and regenerate 

```suggestion
description: The ExtractIt plugin has a collection of actions used to extract various information from text. These include URLs, domains, emails, IPs, and more
```